Dockerfile

```diff
@@ -27,20 +27,35 @@ RUN apt-get update && \
     apt-get autoclean -y && \
     rm -rf /var/lib/apt/lists/*
 
+# --------------------------------------
 # Install TeX Live
 # ref: https://tug.org/texlive/quickinstall.html, https://github.com/Paperist/texlive-ja/blob/main/debian/Dockerfile
+# --------------------------------------
+
+# Set ARG and ENV for installation of TeX Live
+# `TEXLIVE_VERSION` is used on `docker-entrypoint.sh`
+ARG ARCHIVE_URL="https://mirror.ctan.org/systems/texlive/tlnet/install-tl-unx.tar.gz"
+ENV TEXLIVE_VERSION "2023"
```

Comment:
- [x] Update the environment variable `TEXLIVE_VERSION` in `Dockerfile` is updated to the new version

`TEXLIVE_VERSION` was changed from `ARG` to `ENV` to be used in `docker-entrypoint.sh`.